<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EachReport</class>
 <widget class="QWidget" name="EachReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>:/img/border.PNG</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_num">
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_subject">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_time">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="filename">
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>OpenHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>미제출</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="button_upload">
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#button_upload{
	border-image: url(:/res/res/re_b.png);
}

QPushButton#button_upload:hover{
	border-image: url(:/res/res/re_o.png);
}

QPushButton#button_upload:pressed {
	border-image: url(:/res/res/re_c.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>label_3</zorder>
  <zorder>button_upload</zorder>
  <zorder>label_num</zorder>
  <zorder>label_subject</zorder>
  <zorder>checkBox</zorder>
  <zorder>label_time</zorder>
  <zorder>filename</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
